utilities/pipelines/deploymentRemoval/Clear-SubscriptionDeployment.ps1

```diff
@@ -0,0 +1,122 @@
+﻿
+<#
+.SYNOPSIS
+Bulk delete all deployments on the given subscription scope
+
+.DESCRIPTION
+Bulk delete all deployments on the given subscription scope
+
+.PARAMETER subscriptionId
+Optional. The ID of the subscription to remove the deployments from. Defaults to the current context.
+
+.PARAMETER DeploymentStatusToExclude
+Optional. The status to exlude from removals. Can be multiple. By default, we exclude any deployment that is in state 'running' or 'failed'.
+
+.PARAMETER maxDeploymentRetentionInDays
+Optional. The time to keep deployments for beyong the deployment statuses to exclude. In other words, if a deployment is older than the threshold, they will always be deleted.
```

Comment:
```suggestion
Optional. The time to keep deployments with a status to exclude. In other words, if a deployment is in a status to exclude, but older than the threshold, it will be deleted.
```